osu.Server.Spectator/Database/DatabaseAccess.cs

```diff
@@ -659,16 +659,25 @@ public async Task<matchmaking_pool_beatmap[]> GetMatchmakingPoolBeatmapsAsync(ui
             })).ToArray();
         }
 
-        public async Task IncrementMatchmakingSelectionCount(matchmaking_pool_beatmap[] beatmaps)
```

Comment:
Bit weird to remove this but leave the column around. Originally we discussed having one extra column `present_count` so we could get an idea of the ratio of presents vs user selects (where `selection_count` might become a decimal denoting how many users from the full room chose it, or where `present_count` is incremented by `room_size` each present to get a fair statistic).